index.html

```diff
@@ -0,0 +1,157 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css">
+    <link rel="stylesheet" href="style.css">
+    <title>Responisve CV</title>
+</head>
+<body>
+    <main class="container">
+        <aside class="left_side">
+            <div class="profiletext">
+                <div class="imgbox">
+                    <img src="img.jpeg">
+                </div>
+                <h2>ADI SHAIKH</h2>
+                <h3>WEB DEVELOPER</h3>
+            </div>
+            <div class="contactinfo">
+                <h3 class="title">Contact Info</h3>
+                <ul>
+                    <li>
+                        <span class="icon"><i class="fa fa-phone" aria-hidden="true"></i></span>
+                        <span class="text">+91 903 9064 239</span>
+                    </li>
+                    <li>
+                        <span class="icon"><i class="fa fa-envelope-o" aria-hidden="true"></i>
+                        </span>
+                        <span class="text">helloworld@gmail.com</span>
+                    </li>
```

Comment:
You should put these information inside a tag and use tel and mailto in the href attribute for phone number and email respectively so that user can reach you directly by clicking on your phone number and mail....

You can reffer link mentioned below for more info...
 https://www.google.com/amp/s/www.geeksforgeeks.org/how-to-create-mail-and-phone-link-in-html/amp/

Reply:
thanks for sharing, got to know something new.

Reply:
great, you can do same with phone number also put i think you are not referred whole page...
if any one clicks on phone number then the page will redirected to calling app...
as same as mail....